src/finder/codeBlockFinder.ts

```diff
@@ -3,8 +3,8 @@ import { Constants } from '../constants';
 import { CodeFinder, UmlCodeContent } from './finder';
 import { extractSubIncludedText } from './finderUtil';
 
-export class RawFileFinder implements CodeFinder {
-  private readonly URL_REGEX = /^.*\.(plantuml|pu|puml|wsd)(\?.*)?$/;
+export class CodeBlockFinder implements CodeFinder {
+  private readonly URL_REGEX = /(file)|(https?)|([A-Z]):\/\/.+/;
```

Comment:
windows file URI is like `file://C:/.+` NOT `C://.+` ... (really confusing!)
so `/(file)|(https?):\/\/.+/` is fine

Reply:
How about `/(file|https?):\/\/.+/` ?

Reply:
better one!

Reply:
`/(file)|(https?):\/\/.+/.test("file")` yields `true`, so I think this must be `/(file|https?):\/\/.+/`

Reply:
c.f. https://www.boost.org/doc/libs/1_56_0/libs/regex/doc/html/boost_regex/syntax/basic_extended.html#boost_regex.syntax.basic_extended.operator_precedence